src/porklock/config.clj

```diff
@@ -58,10 +58,22 @@
   (when-not (cc/validate-config configs config-valid)
     (throw+ {:error_code ce/ERR_CONFIG_INVALID})))
 
+(defn- set-props-from-string
```

Comment:
I debated adding this to iplant-clojure-commons. I think that porklock will likely be the only place that we need something like this, so I put it here for now.